### ISSUE TITLE
PVR - Mythtv - Feature request

### ISSUE THREAD

**Issue by talcumjz** [koying/SPMC#1177]
Good day. Love SPMC just saying. 

With that is it possible to get the MYTHTV PVR such that it can connect to a MYTHTV 0.29 backend. It will only connect to 0.28 at the moment.

Thanks in advanced,
Wayne

**Comment by martinm76** [comment]
I would very much like to see this as well. I only just tried to connect to my (0.)29 backend today and saw that is was not supported. I expect the protocol haven't changed all that much and it is simply that the version is not recognized? I think they went from 0.28 til 29, right?

**Comment by talcumjz** [comment]
Yes Mythtv went from 0.28 to 0.29. SPMC won't work with .29. Kodi will though.


Sent via the Samsung Galaxy S8+, an AT&T 4G LTE smartphone
-------- Original message --------From: martinm76 <notifications@github.com> Date: 3/20/18  11:20 AM  (GMT-04:00) To: koying/SPMC <SPMC@noreply.github.com> Cc: talcumjz <wberryment@gmail.com>, Author <author@noreply.github.com> Subject: Re: [koying/SPMC] PVR - Mythtv - Feature request (#1177) 
I would very much like to see this as well. I only just tried to connect to my (0.)29 backend today and saw that is was not supported. I expect the protocol haven't changed all that much and it is simply that the version is not recognized? I think they went from 0.28 til 29, right?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/koying/SPMC","title":"koying/SPMC","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/koying/SPMC"}},"updates":{"snippets":[{"icon":"PERSON","message":"@martinm76 in #1177: I would very much like to see this as well. I only just tried to connect to my (0.)29 backend today and saw that is was not supported. I expect the protocol haven't changed all that much and it is simply that the version is not recognized? I think they went from 0.28 til 29, right?"}],"action":{"name":"View Issue","url":"https://github.com/koying/SPMC/issues/1177#issuecomment-374637499"}}}
